### PR TITLE
Add content poilicy page for mozilla.social (Fixes #12865)

### LINKED ISSUE #12865: Content Poilicy page for mozilla.social
## Description

The page will live at https://www.mozilla.org/about/governance/policies/social-content-policies/

---

#### Success Criteria
- [ ] Policy page is live in production

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Adds content policy page for mozilla.social

## Issue / Bugzilla link

#12865

## Testing

http://localhost:8000/en-US/about/governance/policies/social-content-policies/

### CODE REVIEW

**Comment by alexgibson** [comment]
This has been reviewed and approved by legal

File: bedrock/mozorg/templates/mozorg/about/governance/policies/social-content-policies.html [hunk 1]
```diff
@@ -0,0 +1,639 @@
...
+    <li><a href="#self-harm">Self-Harm</a></li>
+    <li><a href="#sexualized-content">Sexualized content</a></li>
+    <li><a href="#child-sexual-abuse">Child Sexual Abuse</a></li>
+    <li><a href="#third-party-violations">Violations of Third Party Privacy</a></li>
+    <li><a href="#misinformation">Misinformation</a></li>
+    <li><a href="#illegal-goods">Promotion of illegal goods</a></li>
+    <li><a href="#fraud-counterfeit-goods">Fraud/Counterfeit goods</a></li>
+    <li><a href="#extremist-content">Terrorist and Violent Extremist Content</a></li>
+    <li><a href="#hate-speech">Hate Speech</a></li>
+  </ul>
```
Comment (robhudson) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/social-content-policies.html:34]: Opening is an `<ol>` but closed with a `</ul>`.

Comment (alexgibson) [reply bedrock/mozorg/templates/mozorg/about/governance/policies/social-content-policies.html]: Nice catch!

**Review by robhudson — APPROVED** [review]
Not a FE-er but looks good to me! 👍

**Review by nathan-barrett — APPROVED** [review]
⚽ +

**Pull request merged**